Source/Lib/Codec/EbEntropyCoding.c

```diff
@@ -4744,7 +4744,7 @@ static void WriteUvlc(
 	EB_U32 numberOfBits = 1;
 	EB_U32 tempBits = ++bits;
 
-	while (1 != tempBits)
+	while (1 != tempBits && tempBits > 0)
```

Comment:
Looks good to me, but it's better to change to "while (tempBits > 1)". :)

Reply:
Thanks for your suggestion. I applied it :)